src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    public static Task[] tasks = new Task[100];
+    public static int tasksLength = 0;
+
+    public static void printNewTask(int taskNumber) {
+        Greeting.printSeperator();
+        System.out.println("\tGot it. I've added this task: \n"+ "\t\t" + tasks[tasksLength].printTask());
+        tasksLength ++;
+        System.out.println("\tNow you have " + tasksLength + " tasks in the list.");
+        Greeting.printSeperator();
+    }
```

Comment:
I like the placing of methods before the switch cases to make the switch cases more readable